packages/main/src/methods/count-tokens.ts

```diff
@@ -22,8 +22,13 @@ export async function countTokens(
   apiKey: string,
   model: string,
   params: CountTokensRequest,
+  requestInit?: RequestInit,
```

Comment:
Can we name this `fetchOptions`? I know the TypeScript type is RequestInit but documentation for `fetch` usually calls this param `options` so users might not realize this is the same set of params usually passed to `fetch`.

Reply:
Yes, ebabefc53fe4123e75e78eb52fe4d808215a32a0